pyccel/ast/core.py

```diff
@@ -11,7 +11,7 @@
 from sympy import sympify
 from sympy import Add as sp_Add, Mul as sp_Mul, Pow as sp_Pow
 from sympy import Eq as sp_Eq, Ne as sp_Ne, Lt as sp_Lt, Le as sp_Le, Gt as sp_Gt, Ge as sp_Ge
-from sympy import Integral, Symbol, Tuple
+from sympy import Integral, Symbol
```

Comment:
I know that this is not strictly related to this PR, but did you know that we were importing `Integral` from `sympy`?? Apparently it only appears in line 286 of this file, where `Derivative` is also used (which is imported on line 24)...

@saidctb Do you know anything about this? Is it used by Psydac?

Reply:
it is used in `allocatable_like`  but I don't think it's used anywhere we can delete it

Reply:
This comment is about issue #663. That issue is for all the extra sympy objects which are imported but used infrequently enough to not require their own PR

Reply:
OK, thanks!